fenix/docs/Glean-Debugging-for-Fenix.md

```diff
@@ -0,0 +1,77 @@
+# Glean Debugging for Fenix
+
+**Purpose:** To be able to test that pings are sending data as expected.
+
+**You need:**
+
+* An Android phone (note: this should work the same for the Android simulator)
+* A cord to plug the android phone into your computer
+* The Firefox app of your choice installed on the phone
+
+## Setup
+
+1. Plug in the Android phone to your computer (or start the Android Simulator).
+
+2. Make sure you have usb debugging enabled on your phone.
+	* Check for Settings > Developer Options
+	* If you don’t see this:
+		* Go to Settings > About Device
+		* Tap the Build Number 7 times
+		* This turns on Developer Mode
+	* Go to Settings> Developer Options
+		* Turn on the enable usb debugging option
+
+3. Check that you have adb installed on your computer.
+	* In the command line, run: `brew install android-platform-tools`
```

Comment:
Homebrew is very MacOS specific, can we generalize this any or just link to the landing page for the android-platform-tools?

Reply:
We probably could, but I won't have the time for that; I'd rather see this landed and updated in follow-ups.